core/kernel/system_spec.rb

```diff
@@ -49,13 +49,20 @@
   end
 
   it "does not expand shell variables when given multiples arguments" do
-    lambda { @object.system("echo", @shell_var) }.should output_to_fd("#{@shell_var}\n")
+    platform_is_not :windows do
+      lambda { @object.system("echo", @shell_var) }.should output_to_fd("#{@shell_var}\n")
+    end
+    platform_is :windows do
+      # TODO: but on Windows it does - fix this once there's clarity on https://bugs.ruby-lang.org/issues/12231
+      lambda { @object.system("echo", @shell_var) }.should output_to_fd("foo\n")
+    end
```

Comment:
Same here, it's best to split in two examples. BTW the spec description should say "does expand" since that's the behavior. (and of course keep the reference to the bug, thanks for reporting it!)